src/styles/sidebar/components/PaginationNavigation.scss

```diff
@@ -0,0 +1,50 @@
+@use '../../mixins/buttons';
+@use "../../mixins/layout";
+@use "../../mixins/utils";
+@use "../../variables" as var;
+
+.PaginationNavigation {
+  @include utils.font--large;
+  @include layout.row;
+
+  &__relative {
+    // Set a minimum width for the previous and next button containers so that
+    // there is no jumping around as they appear and disappear.
+    min-width: 5em;
+  }
+
+  &__next {
+    // Make sure "next" button is flush right
+    @include layout.row($justify: flex-end);
+  }
+
+  &__pages {
+    @include layout.row($justify: center, $align: center);
+    flex-grow: 1;
+  }
+
+  &__button,
+  &__page-button {
+    @include buttons.button--labeled(
+      $background-color: transparent,
+      $active-background-color: var.$grey-3
+    );
+  }
+
+  &__page-button {
+    margin: var.$layout-space--xxsmall;
+    padding: var.$layout-space--small var.$layout-space;
+  }
+
+  &__button-right {
+    // TODO TEMPORARY adjustment until we can adjust the actual icon
```

Comment:
What is the issue here?

Reply:
This comment is poorly-worded and actually inaccurate, so thanks for calling out.

The issue is simply that the styling for labeled-buttons-with-icons assume that the icon is to the left of the label and sets the margins on the SVG icon accordingly. This doesn't look right when the icon's on the right.

This needs to be fixed correctly in `Button` styling—we're _imminently_ working on `Button` components, so I'd ask for permission to leave this TODO (I've converted to `FIXME`) in place for the next ~week.